.gitlab-ci.yml

```diff
@@ -101,7 +116,35 @@ test_torch_agents:
       - torch_agents/htmlcov
     expire_in: 1 week
 
+validation_test:on-schedule:
+  timeout: 7h 30m
+  rules:
+    - if: $NIGHTLY_BUILD == "ENABLED"
+  image: docker:19.03.1
+  services:
+    - docker:19.03.1-dind
+  variables:
+    # Use TLS https://docs.gitlab.com/ee/ci/docker/using_docker_build.html#tls-enabled
+    DOCKER_HOST: tcp://docker:2376
+    DOCKER_TLS_CERTDIR: "/certs"
+  stage: validation
+  before_script:
+    - apk update
+    - apk add curl bash docker-compose make automake gcc g++ subversion python3-dev musl-dev postgresql-dev nginx gettext apache2-utils
+    - curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Comment:
To make sure the docker-compose version that gets called is the right one you should run `docker-compose -v` somewhere in there.

Also remove the one installed by `apk` above to avoid one _shadowing_ the other.